comms-templates/vulnerability-announcement-email.md

```diff
@@ -7,12 +7,17 @@ issue](vulnerability-announcement-issue.md)._
 
 TO: `kubernetes-announce@googlegroups.com, dev@kubernetes.io, kubernetes-security-announce@googlegroups.com, kubernetes-security-discuss@googlegroups.com, distributors-announce@kubernetes.io, kubernetes+announcements@discoursemail.com`
 
+Tip: Make sure you are a member / manager of these groups before sending the email as posting new topics is restricted
+to members.
```

Comment:
who do we contact to be added?

Reply:
We can add ourself by just clicking "JoinList" to be a member.

Reply:
During the onboarding process the existing SRC members will make you an owner of some groups and a manager of another, but the dev@ list requires you to join it yourself.